app/controllers/wishlists_controller.rb

```diff
@@ -0,0 +1,22 @@
+class WishlistsController < ApplicationController
+  def new
+    @wishlist = Wishlist.new
+  end
+
+  def create
+    @wishlist = Wishlist.new(wishlist_params)
+    @wishlist.user_id = current_user.id
+    if @wishlist.save
+      flash[:notice] = "Success"
+      redirect_to dashboard_path
```

Comment:
Consider consolidating with the line above via `redirect_to dashboard_path, notice: "Success"`